test/caption-stream-test.js

```diff
@@ -232,43 +252,48 @@
   });
 
   test('backspaces are applied to non-displayed memory', function() {
-    var captions = [];
+    var captions = [], packets;
     cea608Stream.on('data', function(caption) {
       captions.push(caption);
     });
 
-    // RCL, resume caption loading
-    cea608Stream.push({ ccData: 0x1420 });
-    // '01'
-    cea608Stream.push({
-      ccData: characters('01')
-    });
-    // backspace
-    cea608Stream.push({ ccData: 0x1421 });
-    cea608Stream.push({
-      ccData: characters('23')
-    });
-    // EOC, End of Caption
-    cea608Stream.push({ pts: 1 * 1000, ccData: 0x142f });
-    // EOC, End of Caption
-    cea608Stream.push({ pts: 3 * 1000, ccData: 0x142f });
+    packets = [
+      // RCL, resume caption loading
+      { ccData: 0x1420 },
+      // '01'
+      { ccData: characters('01') },
+      // backspace
+      { ccData: 0x1421 },
+      { ccData: characters('23') },
+      // EOC, End of Caption
+      { pts: 1 * 1000, ccData: 0x142f },
+      // RCL, resume caption loading
```

Comment:
Might be worth a comment on why this control code is necessary between the two EOCs.